templates/csv.yaml

```diff
@@ -685,12 +685,12 @@ spec:
   relatedImages:
     - name: knative-operator
       # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-      image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:openshift-knative-operator
+      image: registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0:openshift-knative-operator
     - name: knative-openshift
       # This reference will be replaced in local builds and CI via hack/lib/catalogsource.bash.
-      image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:knative-operator
+      image: registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0:knative-operator
```

Comment:
```
docker pull registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0:knative-operator
Error: initializing source docker://registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0:knative-operator: reading manifest knative-operator in registry.ci.openshift.org/openshift/openshift-serverless-v1.23.0: name unknown
```

Reply:
looks like they are not really there, yet - did the actual version build/promotion happen?

Reply:
Wtf? There was a slight delay in promotion, but I was able to pull all of them locally last week.

Reply:
Well, maybe that should be better. https://github.com/openshift-knative/serverless-operator/pull/1586/commits/6440ce15b5e42475dd5026299b2d568136072ba2